codegen-winter/src/imports.rs

```diff
@@ -3,16 +3,18 @@ use super::Scope;
 /// Adds the required imports to the provided scope.
 pub(super) fn add_imports(scope: &mut Scope) {
     // add winterfell imports
-    scope.import(
-        "winter_air::TransitionConstraintDegree",
-        "TransitionConstraintDegree",
-    );
-    scope.import("winter_air::TraceInfo", "TraceInfo");
-    scope.import("winter_air::ProofOptions", "WinterProofOptions");
-    scope.import("winter_air::EvaluationFrame", "EvaluationFrame");
-    scope.import("winter_air::Assertion", "Assertion");
-    scope.import("winter_air::AirContext", "AirContext");
-    scope.import("winter_air::Air", "Air");
+    scope.import("winter_air", "TransitionConstraintDegree");
+    scope.import("winter_air", "TraceInfo");
+    scope.import("winter_air", "ProofOptions as WinterProofOptions");
+    scope.import("winter_air", "EvaluationFrame");
+    scope.import("winter_air", "Assertion");
+    scope.import("winter_air", "AirContext");
+    scope.import("winter_air", "Air");
```

Comment:
nit: since the rest of it is nicely alphabetized, I would put these 3 below the "AuxTraceRandElements" import